### PR TITLE
Severity statistics and csv export

### LINKED ISSUE #1104: Showing severity report count at the statistics page

### PR DESCRIPTION
**Pull request by csordasmarton**
- Showing severity report count at the statistics page.
- Statistics can be exported to CSV.

cc_sev_statistics

Closes #1104

### CODE REVIEW

**Comment by whisperity** [comment]
The button feels weird for me. It could be that the alignment of the text is off a few pixels, or the colour is too saturated.

**Comment by csordasmarton** [comment]
@whisperity You have good eyes to catch pixel alignment problems :smile: I have fixed this and I have also changed the color of the export button:

cc_statistics_export

What do you think?

File: www/scripts/codecheckerviewer/CheckerStatistics.js [hunk 1]
```diff
@@ -336,6 +475,25 @@ function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
...
+      }, parent.domNode);
+
+      dom.create('span', {
+        class : 'title',
+        innerHTML : title
+      }, wrapper);
+
+      dom.create('a', {
+        class : 'export-to export-to-csv',
+        innerHTML : 'Export: CSV',
```
Comment (whisperity) [inline www/scripts/codecheckerviewer/CheckerStatistics.js]: Looking good now. I think we could omit the `:` here.

**Comment by whisperity** [comment]
I think this feature (checker stats) is `CodeChecker cmd sum` which does have an `--output <format>` option.

**Pull request merged**